src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -57,35 +65,31 @@
     },
     {
       "what": [
-        "new page block type for Geometry",
-        "Geometry page block renders a map within the rendering constraint from the Geometry configurator with points/markers and polygons.",
-        "Geometry configurator loads data from the configured feed sources."
+        "geometry page block to render a map with data points collected from record sources.",
+        "The geometry page block can show either points, lines, or polygons based on the configured record feed sources."
       ],
       "why": [
-        "user can now configure a feed source for the Geometry which then loads the data in the record page, which is now rendered in markers and polygons."
+        "to allow users the ability to configure a source for their geometry record which then can be shown on the map as a point, line or a polygon depending on the configuration."
```

Comment:
This doesn't sound like a reason to me; perhaps something in the lines of enabling users to visualize their things would sound better?